tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -22,6 +22,7 @@ var _ = Describe("odo devfile catalog command tests", func() {
 		// odo catalog list components.
 		// TODO: Investigate this more.
 		helper.Cmd("odo", "preference", "set", "registrycachetime", "0").ShouldPass()
+		helper.SetDefaultDevfileRegistryAsStaging()
```

Comment:
Why have you explicitly set this for this test and the one below? Why not for all?

Reply:
It was originally on the BeforeSuite, but it should be executed after helper.CommonBeforeEach(), because we set GLOBALODOCONFIG in it. 
So we need to execute it in each top level BeforeEach of the `devfile` directory

Reply:
Because CommonBeforeEach is called for all tests, not only the ones related to devfiles. There would probably not be a problem, but I would prefer not to.